README.developers.md

```diff
@@ -341,14 +341,131 @@ $ ./run_reg_test.py odin_reg_strong
 ```
 and should be used when making changes to Odin.
 
-## Unit Tests
+# Unit Tests
 
 VTR also has a limited set of unit tests, which can be run with:
+
 ```shell
 #From the VTR root directory
 $ make && make test
 ```
 
+This will run `test_vtrutil`, `test_vpr`, and `test_fasm`. `test_archfpga` is not run.
```

Comment:
Could you also add a small comment on CMake's `add_test` feature here. See:
- https://cmake.org/cmake/help/latest/command/add_test.html
- https://cmake.org/cmake/help/book/mastering-cmake/chapter/Testing%20With%20CMake%20and%20CTest.html

Nothing huge, just a sentence or two explaining that we use this in VTR. Its just not obvious that we make use of it!